src/features/classification/ClassifiedBadge.js

```diff
@@ -1,26 +1,47 @@
 // @flow
 import * as React from 'react';
 
+import PlainButton from '../../components/plain-button';
 import Tooltip from '../../components/tooltip';
 import IconSecurityClassification from '../../icons/general/IconSecurityClassification';
 import SecurityBadge from '../security';
 import { bdlYellorange } from '../../styles/variables';
 import type { Position } from '../../components/tooltip';
+import './ClassifiedBadge.scss';
 
 type Props = {
     name: string,
+    onClick?: () => void,
     tooltipPosition?: Position,
     tooltipText?: string,
 };
 
-const ClassifiedBadge = ({ name, tooltipPosition = 'bottom-center', tooltipText }: Props) => (
-    <Tooltip isDisabled={!tooltipText} position={tooltipPosition} text={tooltipText} isTabbable={false}>
+const ClassifiedBadge = ({ name, onClick, tooltipPosition = 'bottom-center', tooltipText }: Props) => {
+    const isClickable = typeof onClick === 'function';
+    const isTooltipDisabled = !tooltipText;
+    const badge = (
         <SecurityBadge
             className="bdl-ClassifiedBadge"
-            message={name}
             icon={<IconSecurityClassification color={bdlYellorange} height={10} width={10} strokeWidth={3} />}
+            message={name}
         />
-    </Tooltip>
-);
+    );
+    return (
+        <Tooltip isDisabled={isTooltipDisabled} isTabbable={false} position={tooltipPosition} text={tooltipText}>
+            {isClickable ? (
+                <PlainButton
+                    className="bdl-ClassifiedBadge-editButton"
+                    data-resin-target="editclassification"
```

Comment:
I thought we were using camel case for these

Reply:
Hmm, unsure but just using what we had before